Dockerfile

```diff
@@ -20,6 +20,8 @@ RUN pip install -r base.txt && \
 
 # Install node dependencies
 COPY ./package-lock.json ./package.json /app/
+# TODO: see if we can upgrade webpack/uglify to remedy NODE_OPTIONS
+ARG NODE_OPTIONS=--openssl-legacy-provider
```

Comment:
(For posterity) omitting the `--openssl-legacy-provider` node option results in an error message like this when doing an `npm i`:

```
ERROR in main-6d8044f3191ec0e46aaa.js from UglifyJs
19.84 Error: error:0308010C:digital envelope routines::unsupported
19.84     at new Hash (node:internal/crypto/hash:68:19)
19.84     at Object.createHash (node:crypto:138:10)
19.84     at /app/node_modules/uglifyjs-webpack-plugin/dist/index.js:186:40
19.84     at Array.forEach (<anonymous>)
19.84     at UglifyJsPlugin.optimizeFn (/app/node_modules/uglifyjs-webpack-plugin/dist/index.js:126:142)
19.84     at AsyncSeriesHook.eval [as callAsync] (eval at create (/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:21:1)
19.84     at AsyncSeriesHook.lazyCompileHook (/app/node_modules/tapable/lib/Hook.js:154:20)
19.84     at /app/node_modules/webpack/lib/Compilation.js:1409:36
19.84     at eval (eval at create (/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:11:1)
19.84     at /app/node_modules/copy-webpack-plugin/dist/index.js:598:9
19.84 
19.84 ERROR in vendor-6d8044f3191ec0e46aaa.js from UglifyJs
19.84 Error: error:0308010C:digital envelope routines::unsupported
19.84     at new Hash (node:internal/crypto/hash:68:19)
```

The underlying issue seems to have to do with webpack v4 using an outdated hash function, which has been disallowed as of Node v17. 

See also:

- https://nodejs.org/en/blog/release/v17.0.0
- https://stackoverflow.com/a/73465262 
- https://github.com/webpack/webpack/issues/14532

Reply:
Awesome! Thanks for adding context/reference to the `--openssl-legacy-provider`